src/main/java/com/internet/shop/controllers/ShoppingCartController.java

```diff
@@ -23,6 +23,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         ShoppingCart currentShoppingCart = shoppingCartService.getByUserId(USER_ID);
         List<Product> products = currentShoppingCart.getProducts();
         req.setAttribute("allProducts", products);
+        req.setAttribute("cartId", currentShoppingCart.getId());
```

Comment:
why do you need this ? You can use shoppingCartService.getByUserId(USER_ID) in each controller. And you will always have the id of current user(for now it's USER_ID = 1).

Reply:
Sure. Done